RecoTracker/CkfPattern/plugins/GroupedTrajCandLess.h

```diff
@@ -31,8 +31,9 @@ class dso_internal GroupedTrajCandLess
   template<typename T>
   float score (const T & t) const
   {
-
-    return t.chiSquared()-t.foundHits()*bonus+t.lostHits()*penalty 
+   auto bb = (t.dPhiCacheForLoopersReconstruction()==0 && t.foundHits()>8) ? 2*bonus : bonus; //extra bonus for long tracks not loopers
+    if ( t.lastMeasurement().updatedState().globalMomentum().perp2() < 0.81f ) bb*=0.5f; 
+    return t.chiSquared()-t.foundHits()*bb+t.lostHits()*penalty 
```

Comment:
in TRK meeting I mentioned that the chi2 relatively loses its contribution now.
If previously one hit was worth 5 units of chi2, it is now ranked as 20.
E.g. a 10-dof candidate with perfectly normal chi2 (10) will lose to a 11-dof cand with chi2=29.9 which is ~6 sigma off normal. (old version would have the 11-dof cand only at most 1.2 sigma off to win the score).

If the goal was to push found hits score over the lost hits, my suggestion was to scale down the penalty and bonus to keep the relative weight of chi2 to be about the same.